src/theme.tsx

```diff
@@ -46,43 +46,22 @@ export const kurtTheme = createTheme({
       letterSpacing: '0.46px',
     },
   },
-  palette: {
-    director_producer: {
-      main: '#4BAEFF',
-    },
-    director_of_photography: {
-      main: '#CA9FFF',
-    },
-    series: { main: '#FFB74D' },
-    animation: { main: '#F8CAFF' },
-    documentary: { main: '#79DFC1' },
-    background: {
-      default: '#343A40',
-    },
-  },
   components: {
     MuiAppBar: {
       defaultProps: {
-        sx: {
-          backgroundColor: '#2D2D2D',
-        },
+        sx: { backgroundColor: '#191B1B' },
       },
     },
     MuiCard: {
       defaultProps: {
         elevation: 0,
-        sx: {
-          backgroundColor: '#212529',
-          overflow: 'visible',
-        },
+        sx: { backgroundColor: '#0C0D0D' },
```

Comment:
Should we set any of these colors to variables somewhere so they are easier to reuse? And so we can look at the names and know what color it is quicker?

Reply:
Makes sense, ya